lua/no-neck-pain/config.lua

```diff
@@ -328,6 +328,16 @@ NoNeckPain.options = {
             filetype = nil,
         },
     },
+    callbacks = {
```

Comment:
```suggestion
    -- Allows you to provide custom code to run before (pre) and after (post) no-neck-pain steps (e.g. enabling).
    ---@type table
    callbacks = {
```

Could you also update the README please? It still states `mappings` for this key and doesn't default to `nil`

Reply:
Done.